### ISSUE TITLE
[Accessibility] Indicate success or failure label in the workflow view

### ISSUE THREAD

**Issue by jooyoungseo** [github/vscode-github-actions#379]
Type: <b>Feature Request</b>

For screen reader users, please add label to each workflow item regarding whether the current workflow run has succeeded or faled.

There is no accessible indicator as of this writing.

Extension version: 0.26.2
VS Code version: Code - Insiders 1.93.0-insider (e1db90f81da28c0994b5d50dc2b51e964359e62d, 2024-08-21T05:04:11.586Z)
OS version: Windows_NT x64 10.0.22631
Modes:
